setup.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; -*-
+
+import os
+
+from setuptools import setup
+
+
+def read(path):
+    with open(os.path.join(os.path.dirname(__file__), path)) as f:
+        return f.read()
+
+
+long_description = read("README.md")
+
+setup(
+    name="check-synology",
+    version="1.0.0",
+    url="https://github.com/wernerfred/check_synology",
+    author="Frederic Werner",
+    author_email="wernerf@fim.uni-passau.de",
+    description="Check different values on your Synology DiskStation",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    platforms=["any"],
+    license="AGPL-3.0",
```

Comment:
I deliberately chose this license for you ;]. Please advise if you consider to use a different one.

Reply:
I don't care too much tbh. I want people to use this scripts and am happy if the community reports back useful information or contributes to the project. If someone would "steal" this script i would be disappointed but life goes on anyways. 
So AGPL-3.0 would require everyone to report back or publish own modifications but in the end it is only a License file and i have no real chance to keep track of the distribution. It's more like a conscience thing.

Am i right to assume that providing a license is mandatory for the `setup.py`?

Reply:
It's not so much about a "theft" thing. It's just that a license makes the project complete on this aspect. Everyone will know under which conditions to use the software. Enforcing it is a completely different thing, right ;].

> It's more like a conscience thing.

Exactly. And a matter of completeness.

> Am i right to assume that providing a license is mandatory for the `setup.py`?

If this package will ever be uploaded to PyPI, it will be mandatory. I think it is a good thing when we take the decision right now if you don't have any objections.

Reply:
I have not, do it